week08_pomdp/practice_pytorch.ipynb

```diff
@@ -410,10 +429,13 @@
     "  $$ H = - {1 \\over T} \\sum_t \\sum_a {\\pi(a|s_t) \\cdot \\log \\pi (a|s_t)}$$\n",
     "  * If we __maximize__ entropy we discourage agent from predicting zero probability to actions\n",
     "  prematurely (a.k.a. exploration)\n",
-    "  \n",
-    "  \n",
+    "\n",
+    "- **Policy Loss** \n",
+    "<center>\n",
+    "<img src=\"https://www.informit.com/content/images/chap6_9780135172384/elementLinks/06eqa21_alt.jpg\" width=500px/>\n",
```

Comment:
This looks so out-of-place here. The notation is different. There are no comments about the relationship of this formula to anything else.

Please either make this formula blend in the surrounding text or remove it.

Reply:
Weirdly, what's wrong with the formula??